use different logback configuration in developer mode

inkuss opened the pull request:
- wird die Play Anwendung im Entwicklungsmodus gestartet, wird eine andere Logback Konfiguration geladen
- benutzt JoranConfigurator, um die Konfig Datei zu laden
- mit beispielhafter Konfig-Datei "logback.developer.xml"
  - macht kürzere Meldungen
  - Angabe von Klasse und Zeilennummer im Log
  - die hier mitgelieferte Konfig-Datei kann individuell verbessert oder verfeinert werden
    (z.B. unterschiedl. Log-Level in versch. Log-Dateien o.ä., je nach Belieben).

inkuss commented:
Die Änderung habe ich auf edoweb-test ausgerollt.
Sie ist dort aber wirkungslos, das edoweb-test im Prod-Modus hoch fährt.
Eigentlich müssen Entwicklungen, die nur im Entwicklungsmodus laufen sollen, nur auf den Entwicklungsrechnern getestet werden. Auf Prod/Test nur soweit, dass dort hinerher noch alles so läuft wie vorher (das ist auf edoweb-test der Fall).

jschnasse commented:
Hi Heinzelmann,

bitte nochmal mit dem master abgleichen, wegen der Formatierung. Ich würde auch vorschlagen, den Code für den Logger in eine eigene Klasse auszulagern, damit Globals nicht so groß wird.

Bestes
Jan

inkuss committed:
use different logback configuration in developer mode
     - wird die Play Anwendung im Entwicklungsmodus gestartet, wird eine andere Logback Konfiguration geladen
     - benutzt JoranConfigurator, um die Konfig Datei zu laden
     - mit beispielhafter Konfig-Datei "logback.developer.xml"
       - macht kürzere Meldungen
       - Angabe von Klasse und Zeilennummer im Log
       - die hier mitgelieferte Konfig-Datei kann individuell verbessert oder verfeinert werden
         (z.B. unterschiedl. Log-Level in versch. Log-Dateien o.ä., je nach Belieben).

inkuss committed:
put logger configuration loading in new class DevLoggerContext

jschnasse commented:
Hallo Ingolf,
ich möchte noch herausfinden, warum im application.log, die Messages dreifach geloggt werden. Wenn du noch Kapazitäten hast, könntest Du das auch übernehmen. Lass uns morgen mal besprechen.
